web_sdk/sdk_rewriter.dart

```diff
@@ -28,10 +28,9 @@ import 'dart:_engine' as engine;
   <String>[
 r'''
 export 'src/engine.dart'
-    show persistedPictureFactory, houdiniPictureFactory, platformViewRegistry;
 ''',
 r'''
-export 'dart:_engine' show persistedPictureFactory, houdiniPictureFactory, webOnlyInitializeEngine;
+export 'dart:_engine'
```

Comment:
I would recommend keeping this export as locked down as possible

Reply:
Agreed, but I think it's confusing that the exports list is totally overwritten in the rewriter. We should keep the `dart:_engine` exports limited, but the source of truth should be in `lib/ui.dart`